### ISSUE TITLE
Bug in loading calendar

### ISSUE THREAD

**Issue by marioja** [Kozea/Radicale#1186]
[bad.ics](https://github.com/marioja/Radicale/raw/master/bad.ics)
[good.ics](https://github.com/marioja/Radicale/raw/master/good.ics)

I run the latest radicale 3.0.6 and I get no errors loading good.ics but I get a "Multiple main components" error with the file bad.ics.  I do not believe the bad.ics file is incorrect.  Can you confirm?

**Comment by marioja** [comment]
You are welcomed to use the test files in test scripts if you need them.

**Comment by marioja** [comment]
I will also add that outlook on windows 10 will generate files similar to bad.ics.  This makes radicale incapable of loading them.

**Comment by cRoCx** [comment]
I had this issue importing an .ics-file, which I exported from Thunderbird. Turned out Thunderbird did somehow duplicate a VEVENT in the past. So I had two identical VEVENTs (also same UID) in my .ics-file. This triggered the error message above.
Unfortunately the debug messages of Radicale don't help here. I was able to find the problem after doing the following change in the item/filter.py:
`-                    raise ValueError("Multiple main components")`
`+                    raise ValueError("Multiple main components. Got comp: {}".format(comp))`
This will pinpoint the troublesome VEVENT
